src/app/containers/BulletedListItem/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import { arrayOf, shape, string, oneOfType, bool, oneOf } from 'prop-types';
+import Blocks from '../Blocks';
+import paragraph from '../Paragraph';
+
+const componentsToRender = { paragraph };
+
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+
+const BulletedListItemContainer = ({ blocks }) => {
+  return (
+    <li>
```

Comment:
To reinstate the list semantics (which is removed as a result of a bug with voiceover due to list style none) please add role="listitem" and also role="list" to the ul

Reply:
Updated